src/MongoDB.Driver.Core/Core/Operations/AggregateOperation.cs

```diff
@@ -346,7 +346,7 @@ internal BsonDocument CreateCommand(ConnectionDescription connectionDescription,
 
             if (Feature.AggregateCursorResult.IsSupported(connectionDescription.ServerVersion))
             {
-                var useCursor = _useCursor.GetValueOrDefault(true) || connectionDescription.ServerVersion >= new SemanticVersion(3, 5, 0);
+                var useCursor = _useCursor.GetValueOrDefault(true) || connectionDescription.ServerVersion >= new SemanticVersion(3, 6, 0);
```

Comment:
Version should also be changed here: https://github.com/mongodb/mongo-csharp-driver/blob/2a716e19b631592c31ab4d55e4d03366c65d6be0/src/MongoDB.Driver.Core/Core/Operations/AggregateToCollectionOperation.cs#L270 and here: https://github.com/mongodb/mongo-csharp-driver/blob/2a716e19b631592c31ab4d55e4d03366c65d6be0/tests/MongoDB.Driver.Core.Tests/Core/Operations/AggregateToCollectionOperationTests.cs#L306

Reply:
Done.

Note that these changes to the server version number are very low value. There is no bug and the existing code was working fine.